src/reducers/PagesReducer.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import { DataTablePageReducer } from '../pages/dataTableUtilities/reducer/DataTablePageReducer';
+import getPageInitialiser from '../pages/dataTableUtilities/getPageInitialiser';
+
+/**
+ * Redux reducer controlling the `pages` field.
+ * @param {Object} state  Redux state, `pages` field.
+ * @param {Object} action An action object, following FSA
+ */
+export const PagesReducer = (state = {}, action) => {
+  const { type } = action;
+
+  switch (type) {
+    case 'Navigation/NAVIGATE': {
```

Comment:
This is a type from `react-navigation`, I had a look through and they don't use constants to steal unfortuantely. I agree we need some - and an overall tidy up of all the file structure - wont do it here just because I dont really know where to put it right now

Reply:
👍 .